ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/TestCaseBuilderUtil.java

```diff
@@ -72,15 +72,20 @@ private TestCaseBuilderUtil() {
   public static String buildTestName(
       final Path originalFileName,
       final String testName,
-      final Optional<String> explicitFormat
+      final Optional<String> explicitFormat,
+      final Optional<String> config
   ) {
     final String prefix = filePrefix(originalFileName.toString());
 
     final String pf = explicitFormat
         .map(f -> " - " + f)
         .orElse("");
 
-    return prefix + testName + pf;
+    final String pc = config
+        .map(f -> " - " + f)
+        .orElse("");
+
+    return prefix + testName + pf + pc;
```

Comment:
Does this impact historical names of tests?  (E.g., is there any backwards compatibility that come up with the names in any way?)

FWIW, if the build passes, I'm totally happy.;)

Reply:
Not 100% sure... I hope @vcrfxia can chime in?

My understanding is, that it should be ok, as we run all historic tests no matter their name.

Reply:
It doesn't matter for the historic ones, yeah. It might matter for the latest ones but if that's the case then PlannedTestsUpToDateTest would fail the build and we'd be alerted to it.

Reply:
`PlannedTestsUpToDateTest` did fail and I needed to generate new plans for the newly named tests.

Reply:
Haha -- would have been smart to add the newly generated plan to the commit and lot have the only locally :D

Reply:
Ah-ha!  🤔   Are you really trying to rename all the unit tests to juice your GitHub additions/subtractions?

"Look, I did 10 million lines of code this year...."

Reply:
Busted! 😂